templates/sso-cd-image-stream.json

```diff
@@ -79,6 +79,24 @@
                             "kind": "DockerImage",
                             "name": "registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift:5.0"
                         }
+                    },
+                    {
+                        "name": "6.0",
+                        "annotations": {
+                            "description": "Red Hat Single Sign-On 6 continuous delivery Tech Preview image",
+                            "iconClass": "icon-sso",
+                            "tags": "sso,keycloak,redhat,hidden",
+                            "supports": "sso:7.4",
```

Comment:
I'm not sure about the 7.4 here.

Reply:
It should be `sso:6`. Thanks for spotting this!